textwiser/embeddings/word.py

```diff
@@ -33,6 +33,14 @@ def bytepair_tokenizer(docs, bp):
 
 
 def transformers_loader(pretrained, **kwargs):
+    # Ignore errors not relevant for word embeddings
```

Comment:
interesting!

Reply:
nice history/context logging, btw.

Reply:
@dorukkilitcioglu  @takojunior if this is available in TextWiser.. would we still need that code snippet in mab2rec 2_feat_eng.nb